packages/web/src/app/memos/components/MemoSidebar/SidebarMenuItemAddCategory.tsx

```
@@ -4,14 +4,14 @@ import { useCategoryPostMutation } from '@extension/shared/hooks';
 import { Input } from '@src/components/ui/input';
 import { getSupabaseClient } from '@src/utils/supabase.client';
 import { PlusIcon } from 'lucide-react';
-import { useState } from 'react';
+import { memo, useState } from 'react';
 import { useForm } from 'react-hook-form';
 
 interface Input {
```

Biome

[error] Shouldn't redeclare 'Input'. Consider to delete it or rename it.

'Input' is defined here:


(lint/suspicious/noRedeclare)